### ISSUE TITLE
[COMPONENT] Tooltip

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#3289]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the solution you'd like
We need a way to add additional context to a button or other UI element.
This should be in the form of a Tooltip.
The Tooltip could come in two different modes:

**Normal tooltip:**
Is shown on hover of the parent element and disappers on no-hover.
![Normal-tooltip](https://github.com/kirbydesign/designsystem/assets/71710012/a80ca1d8-78cb-4e42-90cb-22a9b4851302)

**Persistent rich tooltips:**  (feature hint)
Is shown when:
The page loads and a new feature is being explained
Disappers when actively closed (x)
Has option to be shown Inline.
![Rich-tooltip](https://github.com/kirbydesign/designsystem/assets/71710012/b2abaace-9fe5-46ce-9edf-60c7d038581c)

## Have you considered any alternatives?
N/A

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by dennispedersen-bd** [comment]
We need to figure out a good way to describe and document this in relation to the existing feature hint concept. A traditional tooltip is UX-wise conceptually very different from a persistent feature hint banner that points to a button or other component. Let's discuss how we document this.

Feature hint concept has two flavors:
- modal (uses af drawer compont)
- banner (uses persistent tooltip).

An idea is that maybe they should be 2 different showcases.

**Comment by anderslindberg-alt** [comment]
Advisory Core expects to use this feature once it is ready :-) 

ie. to let the user know about keyboard shortcuts for certain functions in the new Theme/menu.

Do we know aprox. when this will be released?

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
